### PR TITLE
Conformation stress elements in convergence tester

### PR DESCRIPTION
**Pull request by bindi-nagda**
This PR updates the convergence tester to include error norm calculations for the stress components, Sxx, Syy and Sxy, in the conformation tensor. The input file has also been updated to reflect the current set of parameters I'm using for doing convergence tests for uniform network volume fraction under four roll mill forcing.

### CODE REVIEW

File: frm/four_roll_mill.cpp [hunk 1]
```diff
@@ -257,16 +260,25 @@ main(int argc, char* argv[])
...
+            {
+                pout << "\nWriting hierarchy data files...\n\n";
+                output_data(patch_hierarchy, ins_integrator, adv_diff_integrator, cf_un_forcing, iteration_num, loop_time, postproc_data_dump_dirname);
+                next_hier_dump_time += viz_hier_time_interval;
+            }
         }
-        // At specified intervals, store hierarchy data for post processing.
-        output_data(patch_hierarchy, ins_integrator, adv_diff_integrator, iteration_num, loop_time, postproc_data_dump_dirname);
+        // pout << "\nWriting hierarchy data files...\n\n";
+        // output_data(patch_hierarchy, ins_integrator, adv_diff_integrator, cf_un_forcing, iteration_num, loop_time, postproc_data_dump_dirname);
```
Comment (abarret) [inline frm/four_roll_mill.cpp:273]: If these are no longer used, then just go ahead and remove them.

File: frm/input2d.cf [hunk 2]
```diff
@@ -1,14 +1,24 @@
-RHO = 1.0
-MAX_MULTIGRID_LEVELS = 6
+RHO = 1.0e-6
+ETA_S = 10.0
+ETA_N = 10.0
+NU_N = 1.0
+NU_S = 1.0
+XI = 0.0
```
Comment (abarret) [inline frm/input2d.cf:6]: Use xi nonzero. We want coupling between the two phases.

File: frm/convergence_tester.cpp [hunk 3]
```diff
@@ -343,11 +386,15 @@ main(int argc, char* argv[])
...
 
+        refine_op = grid_geom->lookupRefineOperator(S_var, "LINEAR_REFINE");
+        refine_alg.registerRefine(S_interp_idx, S_interp_idx, S_scratch_idx, refine_op);
+
         ComponentSelector data_indices;
         data_indices.setFlag(un_scratch_idx);
         data_indices.setFlag(us_scratch_idx);
         data_indices.setFlag(p_scratch_idx);
         data_indices.setFlag(thn_scratch_idx);
+        // data_indices.setFlag(S_scratch_idx);
```
Comment (abarret) [inline frm/convergence_tester.cpp:397]:
```suggestion
        data_indices.setFlag(S_scratch_idx);
```
?

**Review by abarret — APPROVED** [review]
Just some suggestions.

**Comment by bindi-nagda** [comment]
> Just some suggestions.

Just pushed code addressing these.

**Pull request merged**